sympy/solvers/ode/tests/test_single.py

```diff
@@ -1802,53 +1792,81 @@ def _get_examples_ode_sol_2nd_2F1_hypergeometric():
             'func': f(x),
             'examples': {
-    '2nd_nonlinear_autonomous_conserved_01': {
-        'eq': f(x).diff(x, 2) + exp(f(x)) + log(f(x)),
+    '2nd_autonomous_nth_01': {
+        'eq': f(x).diff(x, x) + f(x).diff(x)**2,
+        'sol': [Eq(C1*exp(f(x)), C2 + x)],
+        'simplify_flag': False,
+    },
+    '2nd_autonomous_nth_02': {
+        'eq': f(x).diff(x, x) - f(x).diff(x)**2,
+        'sol': [Eq(-C1*exp(-f(x)), C2 + x)],
+        'simplify_flag': False,
+    },
+    '2nd_autonomous_nth_03': {
+        'eq': f(x).diff(x, x) + f(x).diff(x)**3,
+        'sol': [Eq(C1*f(x) + f(x)**2/2, C2 + x)],
+        'simplify_flag': False,
+    },
+    '2nd_autonomous_nth_04': {
+        'eq': f(x).diff(x, x) - f(x).diff(x)**3,
+        'sol': [Eq(C1*f(x) - f(x)**2/2, C2 + x)],
+        'simplify_flag': False,
+    },
+    '2nd_autonomous_nth_05': {
+        'eq': f(x).diff(x, x) + f(x).diff(x)**4,
         'sol': [
-            Eq(Integral(1/sqrt(C1 - 2*_u*log(_u) + 2*_u - 2*exp(_u)), (_u, f(x))), C2 + x),
-            Eq(Integral(1/sqrt(C1 - 2*_u*log(_u) + 2*_u - 2*exp(_u)), (_u, f(x))), C2 - x)
+            Eq((C1 + 2*f(x))**(S(3)/2)/3, C2 + x),
+            Eq((C1 + 2*f(x))**(S(3)/2)/3, C2 - x)
         ],
         'simplify_flag': False,
     },
-    '2nd_nonlinear_autonomous_conserved_02': {
-        'eq': f(x).diff(x, 2) + cbrt(f(x)) + 1/f(x),
+    '2nd_autonomous_nth_06': {
+        'eq': f(x).diff(x, x) - f(x).diff(x)**4,
         'sol': [
-            Eq(sqrt(2)*Integral(1/sqrt(2*C1 - 3*_u**Rational(4, 3) - 4*log(_u)), (_u, f(x))), C2 + x),
-            Eq(sqrt(2)*Integral(1/sqrt(2*C1 - 3*_u**Rational(4, 3) - 4*log(_u)), (_u, f(x))), C2 - x)
+            Eq(-(C1 - 2*f(x))**(S(3)/2)/3, C2 + x),
+            Eq(-(C1 - 2*f(x))**(S(3)/2)/3, C2 - x)
         ],
         'simplify_flag': False,
     },
-    '2nd_nonlinear_autonomous_conserved_03': {
-        'eq': f(x).diff(x, 2) + sin(f(x)),
+    '2nd_autonomous_nth_07': {
+        'eq': f(x).diff(x, 2) - exp(f(x)) + log(f(x)),
         'sol': [
-            Eq(Integral(1/sqrt(C1 + 2*cos(_u)), (_u, f(x))), C2 + x),
-            Eq(Integral(1/sqrt(C1 + 2*cos(_u)), (_u, f(x))), C2 - x)
+            Eq(Integral(1/sqrt(-2*_v*log(_v) + 2*_v + C1 + 2*exp(_v)), (_v, f(x))), C2 + x),
+            Eq(Integral(1/sqrt(-2*_v*log(_v) + 2*_v + C1 + 2*exp(_v)), (_v, f(x))), C2 - x)
         ],
         'simplify_flag': False,
     },
-    '2nd_nonlinear_autonomous_conserved_04': {
-        'eq': f(x).diff(x, 2) + cosh(f(x)),
+    '2nd_autonomous_nth_08': {
+        'eq': f(x).diff(x, 2) + cbrt(f(x)) + 1/f(x),
         'sol': [
-            Eq(Integral(1/sqrt(C1 - 2*sinh(_u)), (_u, f(x))), C2 + x),
-            Eq(Integral(1/sqrt(C1 - 2*sinh(_u)), (_u, f(x))), C2 - x)
+            [Eq(sqrt(2)*Integral(1/sqrt(-3*_v**(S(4)/3) + 2*C1 - 4*log(_v)), (_v, f(x))), C2 + x),
+            Eq(sqrt(2)*Integral(1/sqrt(-3*_v**(S(4)/3) + 2*C1 - 4*log(_v)), (_v, f(x))), C2 - x)]
         ],
```

Comment:
Here you've made this one a list of lists.

Reply:
Thanks a lot for helping. I hope it passes all tests now.
Thanks.